src/main/java/nl/tudelft/scrumbledore/game/MultiPlayerGameFactory.java

```diff
@@ -0,0 +1,16 @@
+package nl.tudelft.scrumbledore.game;
+
+/**
+ * Creates an instance of a Multi Player Game.
+ * 
+ * @author Niels Warnars
+ */
+public class MultiPlayerGameFactory extends GameFactory {
+
+  
+  @Override
+  Game createGame() {
```

Comment:
Do these methods really need an Override? They're just implementations of abstract methods right?